src/components/consul/selectors/ImageSelector.js

```diff
@@ -109,9 +122,34 @@ export const ImageSelector = ({
   };
 
   const imageSelect = async (imageFunction = selectImage) => {
-    const { uri, type } = await imageFunction();
+    const { uri, type, exif } = await imageFunction();
+    const { GPSLatitude, GPSLongitude } = exif || {};
+
     const { size } = await FileSystem.getInfoAsync(uri);
 
+    const location = {
+      latitude:
+        GPSLatitude || lastKnownPosition?.coords?.latitude || position?.coords?.latitude || 0,
+      longitude:
+        GPSLongitude || lastKnownPosition?.coords?.longitude || position?.coords?.longitude || 0
```

Comment:
please use `position` first and `lastKnownPosition` last, like in other places of the code base, i.e.: `const location = position || lastKnownPosition;` in SueReportLocation. the last known position is described as follows in the expo docs:

> It's considered to be faster than `getCurrentPositionAsync` as it doesn't request for the current location, but keep in mind the returned location may not be up-to-date.

this can be a position, the device has saved earlier somewhen.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/1129/commits/1e3376ec8289d89335d6508806e92b69225ae9f9